Makefile.cross-compiles

```diff
@@ -7,6 +7,8 @@ all: build
 build: app
 
 app:
+	env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -trimpath -ldflags "$(LDFLAGS)" -o ./release/frpc_darwin_arm64 ./cmd/frpc
```

Comment:
Should add if-else case only for go version >= v1.16

Reply:
fixed

Reply:
`sed` options is not compitable with macos.

Maybe you can reset to last version and i will improve it later.

Reply:
``go  version | awk '{print $3}' | sed -r 's|^go([0-9.]+).*$|\1|' | awk -F. '{print $2}'``
tesed on macos 11.2(intel) and 11.2.1(m1)